integrations/PinterestTag/browser.js

```diff
@@ -211,6 +217,7 @@ export default class PinterestTag {
       destEvent?.hasEmptyProducts,
       destEvent?.isUserDefinedEvent
     );
+    pinterestObject.event_id = this.deduplicationKey || messageId;
```

Comment:
Shouldn't this be
`get(message,this.deduplicationKey)`

Reply:
corrected.